src/loggers/crashlytics.ts

```diff
@@ -0,0 +1,8 @@
+export const createCrashlyticsLogger = (crashlytics: any) => (event: string, params: any) => {
+  try {
+    crashlytics.setAttributes(params);
+    crashlytics.recordError(new Error(event));
+  } catch (error) {
+    console.log('Error: Unable to tag firebase crashlytics event:', error);
```

Comment:
Not sure it is a good practice to use console.log in the lib. Maybe we should re-throw the error or at least disable the logs when we are not in DEV mode (or give control of that logs to the project using the library).